openghg/db_config.yaml

```diff
@@ -0,0 +1,8 @@
+config_version: ""
+database:
+  type: "" #tinydb or "mongodb"
+  tinydb:
+    file_path: "tinydb config file"
```

Comment:
We don't need the filepath for the tinydb database as this will be in the object store. I think we'll only need config information for mongodb.